prog/weaver/main.go

```diff
@@ -119,8 +118,12 @@ func main() {
 		os.Exit(0)
 
 	case createDatapath:
-		err := weave.CreateDatapath(datapathName)
-		if odp.IsKernelLacksODPError(err) {
+		err, odp_supported := weave.CreateDatapath(datapathName)
+		if !odp_supported {
+			if err != nil {
+				Log.Fatal(err)
+			}
```

Comment:
Hmm.  No, because we want to exit with the distinguished status code.  Which that code doesn't do - `Fatal` should be `Error`.